docs/fundamentals/apicompat/assembly-validation.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Assembly validation
+description: Learn how assembly validation can be used to develop consistent and well-formed multi-targeting assemblies.
+ms.date: 11/30/2023
+ms.topic: overview
+---
+
+# Assembly validation
+
+Similar to [package validation](/dotnet/fundamentals/apicompat/package-validation/overview), assembly validation tooling allows you, as a library developer, to validate that your assemblies are consistent and well formed. It provides the following checks:
+
+- Validates that there are no breaking changes across versions.
+- Validates that the assembly has the same set of public APIs for all the different runtime-specific implementations.
+- Catches any applicability holes.
+
+You can run assembly validation either as an [MSBuild task](#enable-msbuild-task) or using the [Microsoft.DotNet.ApiCompat.Tool global tool](global-tool.md). You can run assembly validation in an inner or outer build for maximum performance. <!--how?--> It's also fully incremental, meaning the comparison is only triggered if the inputs or outputs have changed.
```

Comment:
How do you run it in inner vs. outer build?

Reply:
Explained that in another comment. It automatically runs in either the outer or inner based on whether the project multi-targets or not.